src/state/graphicalItemsSlice.ts

```diff
@@ -54,6 +54,22 @@ export type CartesianGraphicalItemSettings = {
   hide: boolean;
 };
 
+export type PolarGraphicalItemSettings = {
+  data: ChartData;
+  /**
+   * Each of the graphical items explicitly says which axis it uses;
```

Comment:
Comment says axis but property is dataKey

Reply:
Oops, copy pasta. Will fix later